cmd/kgo-verifier/main.go

```diff
@@ -236,9 +236,14 @@ func main() {
 		workers = append(workers, &srw)
 
 		firstPass := true
-		for firstPass || (len(lastPassChan) == 0 && *loop) {
+		lastPass := false
+		for firstPass || (!lastPass && *loop) {
 			log.Info("Starting sequential read pass")
 			firstPass = false
+			lastPass = len(lastPassChan) != 0
+			if lastPass {
+				log.Info("This will be the last pass")
+			}
```

Comment:
Do I understand correctly the issue to be that before this change, if /last_pass were called after srw.Wait() below, then when execution looped around we would no longer re-enter the loop, and this change allows that final pass through the loop to occur?

Reply:
Correct. Noting to myself to write less abstract commit messages in the future.